dev-notes/xen-in-dlme.md

```diff
@@ -0,0 +1,359 @@
+# Running Xen in DLME
+
+Build Yocto image and flash it to SDcard.
```

Comment:
@arturkow2000 please add simple explaination which OS we are using here and maybe link to other document with meta-fobnail

Reply:
:+1: